lola.html

```diff
@@ -0,0 +1,875 @@
+        in a scope appropriate for account migration, but it can also allow other scopes. The source server may provide
+        filters as an advanced feature.  For example, a server offering both private note-taking and public note-sharing
+        may offer the user a choice of scope - the option to move only public notes to another location.  A server might 
+        also offer features to filter or restrict based on permissions, tags, collections or types. Ideally, the 
+        decision to filter content that a user has asked to be transferred is a decision that the user can make, not one 
+        that the source server imposes on the user, because otherwise content lock-in can be the result of overprotective 
+        filtering.
+      </p>
+
+      <h3>Approaches for moving followers and following</h3>
+
+      <p>
+        The approaches suggested here for moving followers and following lists are entirely different. 
+      </p>
+
+      <p>
+        The user moving their accounts can choose to follow all the same actors in the new location that they followed 
+        in the old location, although there’s no guarantee that all those Follow activities will be successful.  A 
+        destination server can copy the list of accounts to follow and automate this, and they are all brand new Follow 
+        activities created in the new location.  They are not old Follow activities and should not have older timestamps 
+        or breadcrumbs.
+      </p>
+
+      <p>
+        Accounts that follow a moved account may get a notification, via a Move activity, that they have an opportunity
+        to follow a new Actor and stop following the old. They might not get the notification, might ignore it, or might 
+        fail to follow the new account. In any case, any new successful Follow activities from previous followers are brand-new and have no visible or necessary relationship to old Follow activities.
+      </p>
+
+      <p>
+        Timing here is interesting and difficult.  A user may want to set up a new account, check it all out, copy 
+        content over, check all that out, etc.  After the user is satisfied with the move so far they may wish to follow 
+        the same accounts they used to, check THAT out, and finally to advertise the move and disable the old account.  
+        For this reason as well as for flexibility in other use cases, it would be a good idea for destination servers 
+        that choose to implement this entire specification to do so in two parts.  For example, a Web page with a button 
+        to copy content from another account, and a separate button to follow the same accounts as the other account, 
+        would allow the user to follow these steps at leisure or choose to do just the one or the other.  
+      </p>
+
+      <h3>Model for moving shares and likes</h3>
+
+      <p>
+        In the ActivityPub specification, there is a user story involving Chris and Amy:
```

Comment:
This is Christine's deadname. I'd prefer if we use her real name or use different examples.

Reply:
I never thought of Chris and Amy as real people though with Evan in examples I should have known!  Are you updating this example in AP?

Reply:
thank you. I've gone fully cautious and replaced all possible real names and server names with names from my favourite D&D NPC name generator.